File: engine/engine-model/src/main/scala/tools/packet/monster/CatchMonsterFailure.scala
package tools.packet.monster

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class CatchMonsterFailure(private var _message: Int) extends PacketInput {
  def message: Int = _message

  override def opcode(): SendOpcode = SendOpcode.BRIDLE_MOB_CATCH_FAIL
}

File: engine/engine-model/src/main/java/tools/packet/monster/CatchMonsterFailure.java
package tools.packet.monster;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record CatchMonsterFailure(Integer message) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.BRIDLE_MOB_CATCH_FAIL;
   }
}